File: sdk/client-ts/src/___PLAYGROUND___/networks/get-all-networks.ts
import { begin } from '../helpers';

export const getAllNetworks = async () => {
  let context = await begin();

  let networks = await context.client.network.getAll();
  console.log('networks', networks);

  await context.end();
};

getAllNetworks().then((_) => () => console.log('get-all-networks done'));

File: sdk/python/src/__PLAYGROUND__/networks/get-all-networks.py
import asyncio

from __PLAYGROUND__.helper import begin, end


async def getAllNetworks():
    try:
        client, _, _, rrelayer_node = await begin()

        networks = await client.network.getAll()
        print("networks", networks)
    except Exception as e:
        print("getAllNetworks failed:", e)
    finally:
        end(rrelayer_node)


if __name__ == "__main__":
    asyncio.run(getAllNetworks())
    print("get-all-networks done")
